gno.land/pkg/gnoweb/views/funcs.html

```diff
@@ -82,6 +82,16 @@
 <meta name="viewport" content="width=device-width,initial-scale=1" />
 <link rel="stylesheet" href="/static/css/normalize.css" />
 <link rel="stylesheet" href="/static/css/app.css" />
+<link rel="apple-touch-icon" sizes="180x180" href="/static/img/apple-touch-icon.png" />
+<link rel="icon" type="image/png" sizes="32x32" href="/static/img/favicon-32x32.png" />
+<link rel="icon" type="image/png" sizes="16x16" href="/static/img/favicon-16x16.png" />
+<link rel="mask-icon" href="/static/img/safari-pinned-tab.svg" color="#6f6f6f" />
+<meta name="theme-color" content="#ffffff" />
+<meta property="og:title" content="Gno.land - {{.Data.RealmName}}" />
+<meta property="og:url" content="https://gno.land" />
+<meta property="og:image" content="https://gno.land/static/img/og.png" />
+<meta property="og:site_name" content="gno.land" />
+<meta property="og:description" content="{{ .Data.Description }}" />
```

Comment:
Where is this `Description` defined?

Reply:
To be honest, I reused what was already used for description metadata and hadn't tested it yet. But you are right, the content is always empty... Also, in addition to the generic default desc, we should be able (ideally) to have a dynamic desc.

Reply:
What should we write as generic description. Any idea @leohhhn @michelleellen ?

Reply:
Let it empty for now.